pep-3141.txt

```diff
@@ -85,7 +85,7 @@ numbers are supported by this hierarchy. ::
 
         @abstractproperty
         def imag(self):
-            """Retrieve the real component of this number.
+            """Retrieve the complex component of this number.
```

Comment:
It's actually the *imaginary* component. A complex number has a real part and an imaginary part.

Reply:
Thanks for catching this. I will update the change.